dist/images/Dockerfile.kind

```diff
@@ -0,0 +1,9 @@
+ARG K8S_VERSION="v1.18.2"
+
+FROM kindest/node:$K8S_VERSION
+RUN DEBIAN_FRONTEND=noninteractive clean-install \
+    iputils-ping \
+    tcpdump \
+    vim \
+    openvswitch-switch \
+    && systemctl enable openvswitch-switch
```

Comment:
so far we have always used OVS binary that we build off of OVS master branch. Do you know version of OVS is going to be installed on the host? 

I think it should be fine as long as we have the latest OVS release on the host.

Reply:
```
root@ovn-control-plane:/# ovs-vsctl -V
ovs-vsctl (Open vSwitch) 2.13.0
DB Schema 8.2.0
```
kind uses ubuntu 20.04 as base image ... and we use to update the base images to take the new bits of the software

Reply:
@aojea what if someone wants to build their own version of ovs/ovn/modules binaries and in search for an easy way to try out? It would be great to enahnce this PR with optional parameters so that ovs/ovn binaries (as they interdependent) can be built right from the container as well. It would be great to keep it as generic option also, and not just for kind users. Perhaps maintaining it as a separate DS yaml file and make rule. With regards to dkms in container - see how sysdig guys did it as an example. I have such a change sitting in my queue and if we integrate this change, I can rebase my fix to enable builds of ovn/ovs and dkms support on top of it.

Reply:
> wants to build their own version of ovs/ovn/modules binaries and in search for an easy way to try out
this only changes the way we install OVS, instead of installing OVS in a pod inside a container we do it directly in the container. For OVN nothing changes

>  It would be great to enahnce this PR with optional parameters so that ovs/ovn binaries (as they interdependent) can be built right from the container as well

I would keep the build process separate from the addition of components to the cluster node images ... actually is how we do in KIND,[ the node image building is an independent process](https://github.com/kubernetes-sigs/kind/pull/1767)

> Perhaps maintaining it as a separate DS yaml file and make rule. With regards to dkms in container - see how sysdig guys did it as an example. I have such a change sitting in my queue and if we integrate this change, I can rebase my fix to enable builds of ovn/ovs and dkms support on top of it.

It looks to me that you have a plan :smile: but I can not grok all the details just for the comments, I'm happy to help you with that plan if you can provide me more info but I'd like to have this in to unblock this PR https://github.com/ovn-org/ovn-kubernetes/pull/1561. :+1: 

>  I have such a change sitting in my queue and if we integrate this change, I can rebase my fix to enable builds of ovn/ovs and dkms support on top of it.

maybe you can send a PR and we can discuss over it?

Reply:
> > I have such a change sitting in my queue and if we integrate this change, I can rebase my fix to enable builds of ovn/ovs and dkms support on top of it.
> 
> maybe you can send a PR and we can discuss over it?

of course, later in the week. Lets integrate this change and then I can rebase my fix on top so that we have a subsence to discuss.